vms/avm/txs/executor/semantic_verifier.go

```diff
@@ -146,6 +126,53 @@ func (v *SemanticVerifier) ExportTx(tx *txs.ExportTx) error {
 	return nil
 }
 
+func (v *SemanticVerifier) verifyBaseTx(
+	tx *txs.BaseTx,
+	importedIns []*avax.TransferableInput,
+	exportedOuts []*avax.TransferableOutput,
+) error {
+	feeCalculator := fees.Calculator{
+		Config: v.Config,
+	}
+	if err := tx.Visit(&feeCalculator); err != nil {
+		return err
+	}
```

Comment:
moved fee checks from SyntacticVerifier to SematincVerifier, where we will need it to be with dynamic fees